src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -108,14 +119,21 @@ export function QueryPageContentBarChartResultsComponent(props: {
 
     const handleClick = (): void => props.onDownloadChart();
 
-    const colName = props.chartData.column
-    const rowName = props.chartData.row
+    const colName = props.chartData.column;
+    const rowName = props.chartData.row;
 
-    const isChart = colName !== "" || rowName !== ""
+    const isChart = colName !== "" || rowName !== "";
+
+    let chartData = props.chartData.statisticDataAbsolute;
+    let xAxisMax;

```

Comment:
Recommendation: Use `let xAxisMax = undefined` to make it clear that it is not accidentally left undefined in the next part of the code.

Reply:
In this case the linter is complaining ("Do not use useless `undefined`.") I leave it as it is.

Reply:
lol